src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -10,25 +10,26 @@ trait BundleSpecUtils {
   class BundleFooBar extends Bundle {
     val foo = UInt(16.W)
     val bar = UInt(16.W)
-    override def cloneType: this.type = (new BundleFooBar).asInstanceOf[this.type]
   }
   class BundleBarFoo extends Bundle {
     val bar = UInt(16.W)
     val foo = UInt(16.W)
-    override def cloneType: this.type = (new BundleBarFoo).asInstanceOf[this.type]
   }
   class BundleFoo extends Bundle {
     val foo = UInt(16.W)
-    override def cloneType: this.type = (new BundleFoo).asInstanceOf[this.type]
   }
   class BundleBar extends Bundle {
     val bar = UInt(16.W)
-    override def cloneType: this.type = (new BundleBar).asInstanceOf[this.type]
+  }
+
+  class BundleBaz(w: Int) extends Bundle {
+    val baz = UInt(w.W)
+    // This is runtime deprecated, which we test below
```

Comment:
```suggestion
    // This is currently a compiler warning when using the plugin, which we test below.
    // It will be made an error in future releases.
```